dev/breeze/src/airflow_breeze/commands/developer_commands.py

```diff
@@ -357,6 +358,7 @@ def start_airflow(
 @main.command(name="build-docs")
 @click.option("-d", "--docs-only", help="Only build documentation.", is_flag=True)
 @click.option("-s", "--spellcheck-only", help="Only run spell checking.", is_flag=True)
+@argument_packages_plus_all_providers
 @option_builder
 @click.option(
     "--package-filter",
```

Comment:
I think we can change the type of the "--package-filter" to be just 'string` and instead explain in the help that you can use `apache-airflow-providers-<provider>` or `shorthand package id` or `glob pattern matching the full package name`.  There is  no need any more to do auto-complete for those or to display available values - since we have shorthand names which are auto-complete'able and we have the list of those in `--help`` output.

Reply:
Makes sense. Made the change